samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -144,6 +129,78 @@ class DailyCallManager {
     }
   }
 
+  /**
+   * Handles participant-joined, participant-updated, and participant-left events:
+   * - Updates the participant count
+   * - Adds or removes video and audio tracks for the participant based on the event and track state
+   * @param {Object} event - The participant-* event object.
+   */
+  handleNewParticipantState(event) {
+    // Update the participant count
+    this.updateAndDisplayParticipantCount();
+
+    // Get the audio and video tracks and participant ID from the participant-* events
+    const tracks = event.participant.tracks;
+    const participant = event.participant;
+    const participantId = participant.session_id;
+
+    if (event.action === 'participant-left') {
+      if (participant) {
```

Comment:
Two questions:

* Would participant _ever_ be falsy here?  If so, does this mean we could end up with dangling DOM elements that are never removed?
* Is it worth instead checking `if (!participant)` and early-ing out to reduce nesting for readability?